pkg/wallet/wallet.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	// The current version of neo-go wallet implementations.
-	walletVersion = "1.0"
+	walletVersion = "1.1"
```

Comment:
Other nodes would probably choke on this version and we should be compatible (as much as NEP-6 allows to).